app/components/schema-field-radio/component.js

```diff
@@ -0,0 +1,38 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  init() {
+    this._super(...arguments);
+    let key = this.get('key');
+    let document = this.get('document');
+    let defaultValue = this.get('property.default');
+    let initialValue =  '';
```

Comment:
The mechanism of sanitization in schema-field-text / schema-field-select seem simpler to read, any reason to keep this version instead?